lib/tapioca/cli.rb

```diff
@@ -250,6 +249,28 @@ def dsl(*constant_or_paths)
       type: :boolean,
       desc: "Halt upon a load error while loading the Rails application",
       default: true
+    option :annotations,
+      type: :boolean,
+      desc: "Include RBI annotations from remote source",
+      default: true
+    option :annotation_sources,
+      type: :array,
+      default: [CENTRAL_REPO_ROOT_URI],
+      desc: "URIs of the sources to pull gem RBI annotations from"
+    option :annotation_netrc,
+      type: :boolean,
+      default: true,
+      desc: "Use .netrc to authenticate to private annotation sources"
+    option :annotation_netrc_file, type: :string, desc: "Path to .netrc file"
+    option :annotation_auth,
+      type: :string,
+      default: nil,
+      desc: "HTTP authorization header for private annotation sources"
+    option :exclude_annotation,
+      type: :array,
+      banner: "gem [gem ...]",
+      desc: "Excludes annotation for gem while generating RBI",
```

Comment:
Here and all the previous options descriptions, let's pluralize:

```suggestion
      desc: "Excludes annotations for gem while generating RBI",
```